hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -8667,6 +8676,9 @@ public void logAuditEvent(boolean succeeded, String userName,
         }
         sb.append("\t").append("proto=")
             .append(Server.getProtocol());
+        if (cmd.equals(CMD_COMPLETE_FILE) && status != null) {
+          sb.append("\t").append("fileSize=").append(status.getLen());
```

Comment:
we shouldn't only add a new field for this particular command, as it will probably break lots of applications parsing audit log. See https://issues.apache.org/jira/browse/HDFS-9184 for some more context.

Reply:
There are some discussions around compatibility issues with audit logs. @daryn-sharp @kihwal Could you please share your thoughts here?